examples/text_to_image/train_text_to_image.py

```diff
@@ -599,7 +724,6 @@ def collate_fn(examples):
             dirs = sorted(dirs, key=lambda x: int(x.split("-")[1]))
             path = dirs[-1]
         accelerator.print(f"Resuming from checkpoint {path}")
-        accelerator.load_state(os.path.join(args.output_dir, path))
```

Comment:
Why is `resume_from_checkpoint` disabled?

Reply:
Oops, this is removed by mistake.